valuetransfer/src/main/java/nl/tudelft/trustchain/valuetransfer/passport/PassportCaptureActivity.kt

```diff
@@ -48,6 +50,27 @@ open class PassportCaptureActivity : Activity(), TextRecognitionProcessor.Result
             onBackPressed()
         }
 
+        // Override the scanning
+        findViewById<TextView>(R.id.tvScanTitle).setOnClickListener {
+            overrideScan += 1
+            if (overrideScan % 5 == 0) {
+                onSuccess(
+                    MRZInfo(
```

Comment:
Why is this placeholder data here?

Reply:
This is used to override the need to scan a passport or ID and inserts dummy data. As when working on an emulator it is quite difficult to capture a picture. 
This also complements the override already available to skip the NFC check.